apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts

```diff
@@ -0,0 +1,296 @@
+import { Component, Inject, LOCALE_ID, OnInit } from '@angular/core';
+
+import { Options } from 'highcharts';
+
+import {
+  HighstockDataPoint,
+  stockChartOptions,
+} from '@kirbydesign/designsystem/components/highstock-chart/options/highstock-chart-options';
```

Comment:
Ensure is exported from root:
```suggestion
} from '@kirbydesign/designsystem';
```

Reply:
Done.